### DISCUSSION TITLE
Handling colors

### DISCUSSION THREAD

**Discussion by Didas-git** [Didas-git/lilybird#26] [Q&A]
Issue #24 brought to my attention that we currently do not provide any helper to help with handling colors for embeds and proposes a solution.

My question for you is rather simple:
Would you prefer, having a guide that teaches you multiple ways of handling and solving this problem, having a simple helper under `@lilybird/helpers` that provides a simple solution for most people?

Here is my own answer with an added question to the ones who think the same:
We could provide a guide with various ways of handling colors and also redirect people to external libraries like [Tasai](https://github.com/Didas-git/tasai) that can help people with this issue and also give them even more functionality like using it to generate ascii colors for code blocks, but given that, should we also "duplicate" a solution into the helpers package?

Whats your prefered solution to this problem? Feel free to share any idea/issue related to the issue even if it is not pointed out here.

**Comment by Ungolim** [comment]
yes i think having a guide coupled with a simpler helper shipped with lilybird is ideal, and bonus points if we use Tasai under the hood

**Reply by Didas-git** [reply]
Using Tasai under the hood would introduce a dependency that is not really needed in my opinion.
For a simple handler we can easily  provide `parseInt(color.replace("#", ""), 16)`.
But it being this simple is why im not sure it should be a helper, tho to be fair there is already a lot of this simple things for discord markdown helpers so maybe not the best argument one can have.

**Reply by xhyrom** [reply]
I agree to ship simple helper as @Didas-git said and redirect people to external libs if they want something more specific.
